src/tests.rs

```diff
@@ -402,7 +402,7 @@ async fn remove_docs_bad_package_name() {
 #[tokio::test]
 async fn publish_docs_success() {
     let key = "my-api-key-here";
-    let package = "gleam_experimental_stdlib";
+    let package = "gleam_experimental_stdlib_123";
```

Comment:
Why has this been renamed here?

Reply:
to test that a number is allowed (the test code is somewhat involved so was avoiding duplicating the whole test to tack a number on the end)